### PR TITLE
Gdr 2419

### PR DESCRIPTION
**Pull request by gladkia**
# Description
Improve pkgdown site
## What changed?
- updated news and added reference sections
- Related JIRA issue:  GDR-2419
- pkgdown site link: https://gdrplatform.github.io/gDRstyle/

# Logistic checklist
- [X] Package version bumped
- [X] Changelog updated

# Screenshots (optional)

### CODE REVIEW

File: .github/workflows/pkgdown.yaml [hunk 1]
```diff
@@ -62,7 +62,7 @@ jobs:
             BASE_IMAGE=${{ env.BASE_IMAGE }}
 
       - name: Run pkgdown
-        if: ${{ inputs.IS_SUBDIR  == 'false' }}
+        if: ${{ inputs.IS_SUBDIR  != 'true' }}
```
Comment (j-smola) [inline .github/workflows/pkgdown.yaml:65]: Just out of curiosity - what is the reason for this change?

File: _pkgdown.yml [hunk 2]
```diff
@@ -2,3 +2,16 @@ url: https://gdrplatform.github.io/gDRstyle/
 template:
   bootstrap: 5
 
+reference:
+- title: Check
+  desc:  Check helpers (package and its dependencies)
+- contents:
+  - has_keyword("check")
```
Comment (j-smola) [inline _pkgdown.yml:9]: :+1

File: README.md [hunk 3]
```diff
@@ -1,2 +1,5 @@
 # gDRstyle
 A repository describing standards and templates to be upheld/used by all gDR packages.
+
+# Website
+A package website is available under [this link](https://gdrplatform.github.io/gDRstyle/).
```
Comment (j-smola) [inline README.md:5]: 😃

File: R/dependencies.R [hunk 4]
```diff
@@ -90,6 +91,7 @@ checkDependencies <- function(dep_path,
 #'
 #' @return Character vector of any misaligned package versions between
 #' rplatform \code{dependencies.yaml} and package \code{DESCRIPTION}.
+#' @keywords internal
```
Comment (j-smola) [inline R/dependencies.R:94]: We want to hide this function from public view? 👀

File: NEWS.md [hunk 5]
```diff
@@ -1,117 +1,122 @@
...
 
-## 0.1.3.10 (2023-01-05)
-- fixed WARNINGS and NOTES in check()
+## gDRstyle 0.1.3.11 - 2023-01-26
+* improve dependencies check
 
-## 0.1.3.9 (2023-01-05)
-- the note test is stricter
+## gDRstyle 0.1.3.10 - 2023-01-05
+* fixed WARNINGS and NOTES in check-
```
Comment (j-smola) [inline NEWS.md:119]:
```suggestion
* fix WARNINGS and NOTES in check- 
```

**Review by j-smola — APPROVED** [review]
Could you maybe update "Package installation" section in package vignette since `installAllDeps` is not the function that install gDRstyle package?

**Review by marcin-kam — APPROVED** [review]

File: .github/workflows/pkgdown.yaml (hunk 1 above)
Comment (gladkia) [reply .github/workflows/pkgdown.yaml:65]: It's required to properly support gDRstyle case ('IS_SUBDIR' not defined and equals '').

File: R/dependencies.R (hunk 4 above)
Comment (gladkia) [reply R/dependencies.R:94]: Yes. It's for our internal purposes only.
